packages/common-ethereum/src/project/types.ts

```diff
@@ -28,3 +28,17 @@ export interface EthereumProjectNetworkConfig extends ProjectNetworkConfig {
   chainId?: string;
   subnet?: string;
 }
+
+export enum FlareHandlerKind {
```

Comment:
Can this be `EthereumHandlerKind`?

Reply:
There's already a `EthereumHandlerKind` from `types-ethereum` that is being used here: https://github.com/subquery/subql-ethereum/blob/841d98ffbbc839049dbb0d8d063cd5bac5b7e014/packages/common-ethereum/src/project/models.ts#L91

We need this since we implement interfaces from types-ethereum that uses `EthereumHandlerKind`

Reply:
Maybe I can use something like `EthHandlerKind` or `SubqlEthereumHandlerKind`